content/riak/ts/1.6.0/table-management/per-table-object-expiration.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Configure Per Table Object Expiration"
+description: "Enabling and configuring per-table object expiration for Riak TS."
+menu:
+  riak_ts-1.6.0:
+    name: "Per Table Object Expiration"
+    identifier: "table_mng_per_table_expiry"
+    weight: 300
+    parent: "table_management"
+project: "riak_ts"
+project_version: "1.6.0"
+toc: true
+commercial_offering: true
+version_history:
+  in: "1.6.0+"
+  locations:
+    - [">=1.6.0", "table-management/per-table-object-expiration"]
+aliases:
+    - /riakts/1.6.0/table-management/per-table-object-expiration/
+canonical_link: "https://docs.basho.com/riak/ts/latest/table-management/per-table-object-expiration/"
+---
+
+[ttl]: https://en.wikipedia.org/wiki/Time_to_live
+[global expiry]: /riak/ts/1.6.0/table-management/global-object-expiration/
+[create table]: /riak/ts/1.6.0/table-management/creating-activating/
+[create table with]: /riak/ts/1.6.0/table-management/creating-activating/#using-with
+
+By default, LevelDB keeps all of your data. But Riak TS allows you to configure object expiration (`expiry`) or [time to live (TTL)][ttl] for your data on a [global][global expiry] or per table basis.
+
+Expiration is disabled by default, but enabling it lets you expire older objects to reclaim the space used or purge data with a limited time value.
+
+## Enabling Expiry
+
+To enable object expiry for global or per table use, add the `leveldb.expiration` setting to your riak.conf file:
+
+```riak.conf
+leveldb.expiration = on
```

Comment:
So, and this is just me being confused, this turns expiration on for all tables? Is there a way to enable expiration for a specific table after it has been created?